data/categories.txt

```diff
@@ -126,7 +126,7 @@ ko:1카페|레스토랑|음식
 ja:1カフェ|喫茶|食べ物|飲食|コーヒー|珈琲|お茶|茶|紅茶|食事|軽食
 cs:3kavárna|restaurace|hospoda|jídlo
 sk:3kaviareň|reštaurácia|pohostinstvo|jedlo
-nl:3café|3restaurant|3eten
+nl:3tearoom|3brasserie|3café|3restaurant|3eten
```

Comment:
Is it better to leave restaurant here as a search synonym? When you are hungry, you probably can type restaurant and would be happy to go into a nearest cafe if it is nearby.